docs/index.md

```diff
@@ -457,15 +457,25 @@ example requests that use the fields query parameter:
 ### Collection Size Control Inputs
 
 ListVms also allows controlling the size of the resulting collection with the
-use of the sort, limit and offset parameters. These three parameters can be used
-on either the regular or the LDAP query version of the ListVms endpoint.
+use of the sort, limit, and marker parameters. "sort" and "limit" can be
+used on either the regular or the LDAP query version of the ListVms endpoint.
 
 | Param      | Type   | Description                                         |
 | ---------- | ------ | --------------------------------------------------- |
 | sort       | String | Sort by any of the ListVms inputs (except tags).    |
 | sort.order | String | Order direction. See below                          |
 | limit      | Number | Return only the given number of VMs                 |
-| offset     | Number | Limit the collection starting from the given offset |
+| marker     | String | Limit the collection starting from the given VM     |
```

Comment:
I would just put this on one line, and not bother with redrawing the
previous lines to align vertically.  The way it is now (hard-wrapped
into what is essentially a new table row) seems to render a bit oddly
through the Github markdown renderer.